dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -1194,10 +1194,13 @@ public User setUserPrivilege(@Parameter(hidden = true, name = "user")@Auth User
     @ApiOperation(value = "See OpenApi for details")
     public List<LambdaEvent> getUserGitHubEvents(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
             @QueryParam("offset") Integer offset,
-            @DefaultValue(PAGINATION_LIMIT) @QueryParam("limit") Integer limit) {
+            @DefaultValue(PAGINATION_LIMIT) @QueryParam("limit") Integer limit,
```

Comment:
Ditto about adding the OpenAPI `@Parameter` annotation, you can even use the `PAGINATION_OFFSET_TEXT` and `PAGINATION_LIMIT_TEXT` here